src/engine/input/key_mapping/InputEventItem.cs

```diff
@@ -292,6 +299,7 @@ private void OnRebindButtonPressed()
         WaitingForInput = true;
         button.Text = TranslationServer.Translate("PRESS_KEY_DOT_DOT_DOT");
         xButton.Visible = true;
+        InputManager.RebindingIsActive = true;
```

Comment:
I'd put a blank line before this line to highlight that it is related to a different operation than changing the button status.